manifests/t/TUitTUWien/TUtoolbox/2022.0.0.431/TUitTUWien.TUtoolbox.locale.de-DE.yaml

```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: TUitTUWien.TUtoolbox
+PackageVersion: 2022.0.0.431
+PackageLocale: de-DE
+Publisher: TU Wien - TU.it
+PublisherUrl: https://www.it.tuwien.ac.at/
+PublisherSupportUrl: https://www.it.tuwien.ac.at/services/beratung-und-servicedesk/servicedesk/service-center
+# PrivacyUrl: 
+Author: Christian Beck
+PackageName: TUtoolbox
+PackageUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox
+License: © 2021-2022 TU.it, TU Wien.
```

Comment:
A copyright is not a license; Please update this to reflect the package terms as `Proprietary`, `MIT`, `GPL-2.0`, or whatever the actual software License is